lib/galaxy/datatypes/converters/sam_to_bam.py

```diff
@@ -52,7 +87,12 @@ def __main__():
     # sort sam, so indexing will not fail
     sorted_stderr_filename = os.path.join( tmp_dir, 'sorted.stderr' )
     sorting_prefix = os.path.join( tmp_dir, 'sorted_bam' )
-    cmd = 'samtools sort -o "%s" "%s" > "%s"' % ( unsorted_bam_filename, sorting_prefix, output_filename )
+    # samtools changed sort command arguments (starting from version 1.3)
+    samtools_version = LooseVersion(_get_samtools_version())
+    if samtools_version < LooseVersion('1.0'):
+        cmd = 'samtools sort -o "%s" "%s" > "%s"' % ( unsorted_bam_filename, sorting_prefix, output_filename )
+    else:
+        cmd = 'samtools sort -T "%s" "%s" > "%s"' % ( sorting_prefix, unsorted_bam_filename, output_filename )
```

Comment:
It would be safer to have single quotes around the `%s`, e.g.:
```
cmd = "samtools sort -T '%s' '%s' > '%s'" % ( sorting_prefix, unsorted_bam_filename, output_filename )
```
But this is not a regression, so can be addressed in another PR.